pkg/api/core/v1/models/settings.go

```diff
@@ -21,23 +21,23 @@ import (
 
 // This subsection of models provides structures related to the chart values of applications.
 
-// AppSetting represents the Show Response for a chart value variable
-type AppSetting struct {
+// CVSetting represents the Show Response for a chart value variable
+type CVSetting struct {
```

Comment:
I would avoid using single letter if not really required and clear. `ChartValueSetting` is still fine imo.

Reply:
Done with commit f5b1df5235731c0e44b6a0674c255dc77db412e2